examples/configs/swift/SWIFT.conf

```diff
@@ -0,0 +1,117 @@
+# [FILE PATH OPTIONS]
+
+# Path to where the base folder of where the particle information is. For SWIFT
+# outputs, the FoF group information is also contained within the snapshot, so the 
+# paths should be the same.
+SnapshotPath
+HaloPath
+
+# Base name for the snapshot files, e.g. <SnapshotFileBase>_<snap_nr>.hdf5
+SnapshotFileBase
+
+# Base name for the directories where snapshot subfiles are saved, if applicable.
+# For example, <SnashotDirBase>_<snap_nr>/<SnapshotFileBase>_<snap_nr>.<subfile_nr>.hdf5
+# If there is only one file per snapshot, omit this option.
+SnapshotDirBase
+
+# Base directory of where the HBT catalogues will be saved. The HBT catalogue for each snapshot
+# is saved in its own subdirectory, with the number of subfiles matching the number of MPI ranks.
+SubhaloPath
+
+# [FILE IO OPTIONS]
+# This tells HBT+ that the snapshots are SWIFT-like. DO NOT CHANGE!
+SnapshotFormat  swiftsim
+GroupFileFormat swiftsim_particle_index
+
+# The full range of snapshots that the simulation has. If 10 snapshots exist, this would correspond to
+# 0 and 9, respectively.
+MinSnapshotIndex
+MaxSnapshotIndex
+
+# Which subset of snapshots to analyse in an HBT run. For example, if we only want to analyse every 
+# second snapshot of a ten snapshot run: 1 3 5 7 9
+SnapshotIdList
+MaxConcurrentIO 512
+
+# [PARTICLE PROPERTY OPTIONS]
+# For HYDRO runs, whether gas particles split once they become massive, and if these splits should be 
+# taken into account. If enabled, the particle split scripts NEED to be run (see toolbox/swiftsim).
+ParticlesSplit 1
+
+# [UNIT OPTIONS]
+# The units are loaded automatically from the SWIFT snapshots, and chosen such that the output 
+# has no factors of h.
+MassInMsunh  -1
+LengthInMpch -1
+VelInKmS     -1 
+
+# [GRAVITY SOFTENING OPTIONS]
+# The softening is loaded automatically from the SWIFT snapshots.
+SofteningHalo -1
+MaxPhysicalSofteningHalo -1
+
+# [Simulation Box]
+# The box size is loaded automatically from the SWIFT snapshots.
+BoxSize -1
+PeriodicBoundaryOn 1
+
+# [MISCELLANEOUS OUTPUT OPTIONS]
+# If the internal binding energies of subhaloes, as computed by HBT, should be saved alongside
+# the catalogues.
+SaveBoundParticleBindingEnergies 1
+
+# [Subhalo Unbinding]
+# Keeping here as a reference, but it is not used internally.
```

Comment:
If it's never used then why are we keeping them? Is it just so we remember the code has the capability to do it if we ever need it?

Reply:
I guess I should remove all options that are not required by swift, and leave all of the others in the general example config. 

As to whether to remove, I would like to keep track of these potentially useful variables somewhere...

Reply:
My preference would be for them to be in the example.conf rather than here

Reply:
As mentioned in the above conversation, I have removed all unnecessary options from the SWIFT config file, including this one.